docs/concepts/crds/repository.md

```diff
@@ -64,6 +67,39 @@ Here, we are going to describe the various sections of the `Repository` crd.
 - **spec.wipeOut**
 As the name implies, `spec.wipeOut` field indicates whether Stash operator should delete backed up files from the backend when `Repository` crd is deleted. The default value of this field is `false` which tells Stash to not delete backed up data when a user deletes a `Repository` crd.
 
+- **spec.usagePolicy**
+This lets you control which namespaces are allowed to use the Repository and which are not. If you refer to a Repository from a restricted namespace, Stash will reject creating the respective BackupConfiguration/RestoreSession from validating webhook. You can use the usagePolicy to allow only the same namespace, a subset of namespaces, or all the namespaces to refer to the Repository.
+
+#### Repository `spec.usagePolicy` examples
+If you want your Repository CRD to  allow referencing it only from the same namespace, you can write the `spec.usagePolicy` section like the below:
+```yaml
+spec: 
+  usagePolicy:
+    allowedNamespaces:
+      from: Same
+```
+
+ Here is the `spec.usagePolicy` section of a Repository that allows referencing it from all namespaces,
+```yaml
+spec: 
+  usagePolicy:
+    allowedNamespaces:
+      from: All
+```
+
+ To allow a Repository that can be  referred only from `prod` and `staging` namespaces,
```

Comment:
Here, is an example of `usagePolicy` that allows referencing it from only `prod` and `staging` namespace.